testgrid/config.yaml

```diff
@@ -2457,7 +2457,24 @@ test_groups:
 - name: istio-e2e-cluster_wide-auth
   gcs_prefix: istio-prow/e2e-cluster_wide-auth
   num_failures_to_alert: 1
-# Istio CircleCI
+# Istio CircleCI Presubmits
+- name: circleci-e2e-mixer-presubmit
```

Comment:
you may want to come back and tune the alerting, seeing as the other istio jobs have custom settings here.

Reply:
Those alerts are for postsubmits which are assumed to be stable. Presubmits tend to be noisy and many failures originate from actual failures or even intentional, so perhaps alerts on presubmits might be unnecessary.